tensorflow/compiler/xla/service/hlo_rematerialization.cc

```diff
@@ -843,7 +855,7 @@ int64 RematerializationCost(const HloInstruction* instruction,
     return 0;
   }
 
-  CHECK_GT(memory_reduced, 0);
+  CHECK_GT(memory_reduced, 0) << "The memory reduced is " << memory_reduced;
```

Comment:
This isn't necessary; when CHECK_GT fails, it will print the value of `memory_reduced`.  Same for the other CHECK_GTs.

Reply:
Thans for review, fixed.